src/test/run-pass/binding/pat-ranges.rs

```diff
@@ -8,7 +8,7 @@ mod m {
 }
 
 fn main() {
-    if let NUM1 ... m::NUM2 = 10 {} else { panic!() }
-    if let ::NUM1 ... ::m::NUM2 = 11 {} else { panic!() }
-    if let -13 ... -10 = 12 { panic!() } else {}
```

Comment:
Use `#![allow(...)]` here but also extend the tests to exercise `..=` by duplicating all 3 lines above.